### PR TITLE
Latest localized files from touchdown

### PR DESCRIPTION
**Pull request by jwallra**
Feel free to chime in if any of the translations are incorrect/misleading.

### CODE REVIEW

File: localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="mobile_center_distribute_update_dialog_title">Доступно обновление приложения</string>
+  <string name="mobile_center_distribute_update_dialog_message_optional">Обновление %1$s %2$s (%3$d) доступно для скачивания и установки.</string>
+  <string name="mobile_center_distribute_update_dialog_message_mandatory">Разработчики этого приложения требуют выполнить обновление до %1$s %2$s (%3$d).</string>
```
Comment (MatkovIvan) [inline localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml]: This sounds _much_ worse than https://github.com/Microsoft/mobile-center-sdk-android/blob/develop/sdk/mobile-center-distribute/src/main/res/values-ru/mobile_center_distribute.xml#L5

File: localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <string name="mobile_center_distribute_update_dialog_download">Скачать</string>
+  <string name="mobile_center_distribute_update_dialog_postpone">Спросить через день</string>
```
Comment (MatkovIvan) [inline localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml]: I prefer https://github.com/Microsoft/mobile-center-sdk-android/blob/develop/sdk/mobile-center-distribute/src/main/res/values-ru/mobile_center_distribute.xml#L7

File: localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <string name="mobile_center_distribute_update_dialog_view_release_notes">Просмотреть заметки о выпуске</string>
+  <string name="mobile_center_distribute_dialog_actioned_on_disabled_toast">Распространение отключено</string>
+  <string name="mobile_center_distribute_unknown_sources_dialog_message">В целях безопасности на вашем устройстве настроен запрет на установку приложений, полученных из неизвестных источников.</string>
+  <string name="mobile_center_distribute_unknown_sources_dialog_settings">Параметры</string>
```
Comment (MatkovIvan) [inline localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml]: System settings on Android translated as "Настройки" https://github.com/Microsoft/mobile-center-sdk-android/blob/develop/sdk/mobile-center-distribute/src/main/res/values-ru/mobile_center_distribute.xml#L11

File: localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  <string name="mobile_center_distribute_downloading_mandatory_update">Скачивается обновление</string>
+  <string name="mobile_center_distribute_download_progress_number_format">%1$d из %2$d МБ</string>
+  <string name="mobile_center_distribute_install_ready_title">Обновление для приложения скачано</string>
+  <string name="mobile_center_distribute_install_ready_message">Обновление %1$s %2$s (%3$d) скачано и готово к установке.</string>
+  <string name="mobile_center_distribute_install">Установить</string>
+  <string name="mobile_center_distribute_notification_category">Обновления приложений</string>
```
Comment (MatkovIvan) [inline localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml]: "App" in the plural here. I guess it is one thing.

File: localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="mobile_center_distribute_update_dialog_title">Доступно обновление приложения</string>
```
Comment (MatkovIvan) [inline localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml]: Nitpick: it takes two lines, [shorter version](https://github.com/Microsoft/mobile-center-sdk-android/blob/develop/sdk/mobile-center-distribute/src/main/res/values-ru/mobile_center_distribute.xml#L3) looks better for title

File: localization/mobile-center-distribute/ko-KR/mobile_center_distribute.xml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="mobile_center_distribute_update_dialog_title">앱 업데이트 사용 가능</string>
+  <string name="mobile_center_distribute_update_dialog_message_optional">%1$s %2$s(%3$d)을(를) 다운로드하여 설치할 수 있습니다.</string>
+  <string name="mobile_center_distribute_update_dialog_message_mandatory">이 앱의 개발자가 %1$s %2$s(%3$d)(으)로 업데이트를 요구합니다.</string>
+  <string name="mobile_center_distribute_update_dialog_download">다운로드</string>
+  <string name="mobile_center_distribute_update_dialog_postpone">내일 다시 묻기</string>
+  <string name="mobile_center_distribute_update_dialog_view_release_notes">릴리스 정보 보기</string>
+  <string name="mobile_center_distribute_dialog_actioned_on_disabled_toast">배포를 사용할 수 없음</string>
+  <string name="mobile_center_distribute_unknown_sources_dialog_message">보안상의 이유로, 알 수 없는 원본에서 가져온 앱의 설치를 차단하도록 장치가 설정되어 있습니다.</string>
```
Comment (jaelim-ms) [inline localization/mobile-center-distribute/ko-KR/mobile_center_distribute.xml]: No comma.

File: localization/mobile-center-distribute/ko-KR/mobile_center_distribute.xml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="mobile_center_distribute_update_dialog_title">앱 업데이트 사용 가능</string>
+  <string name="mobile_center_distribute_update_dialog_message_optional">%1$s %2$s(%3$d)을(를) 다운로드하여 설치할 수 있습니다.</string>
+  <string name="mobile_center_distribute_update_dialog_message_mandatory">이 앱의 개발자가 %1$s %2$s(%3$d)(으)로 업데이트를 요구합니다.</string>
+  <string name="mobile_center_distribute_update_dialog_download">다운로드</string>
+  <string name="mobile_center_distribute_update_dialog_postpone">내일 다시 묻기</string>
```
Comment (jaelim-ms) [inline localization/mobile-center-distribute/ko-KR/mobile_center_distribute.xml]: The meaning itself is right but I'd prefer `하루동안 열지 않기` that is used in iOS.

File: localization/mobile-center-distribute/ko-KR/mobile_center_distribute.xml [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  <string name="mobile_center_distribute_update_dialog_view_release_notes">릴리스 정보 보기</string>
+  <string name="mobile_center_distribute_dialog_actioned_on_disabled_toast">배포를 사용할 수 없음</string>
```
Comment (jaelim-ms) [inline localization/mobile-center-distribute/ko-KR/mobile_center_distribute.xml]: `배포` seems not right. This is a name of Distribute so it should be something else but I really don't know.
The translation is anyway wrong. Its meaning is "Cannot use Distribute", not "being disabled". We probably need a human translation for this one.

Comment (jwallra) [reply localization/mobile-center-distribute/ko-KR/mobile_center_distribute.xml]: I'm going to update it as-is unless you want to give a better translation.  Kind of a slippery slope to ask for human translations on some but not others.

**Pull request merged**